### PR TITLE
Added details of synchronization bulk upload API

### PR DESCRIPTION
**Pull request by cmmdesai**
This docs PR is tracked via this work item https://dev.azure.com/msft-skilling/Content/_queries/edit/116745/?triage=true

### CODE REVIEW

**Comment by cmmdesai** [comment]
We can proceed with the content review. Please hold the merge to the main branch as we'd like to time it so it aligns with our public preview announcement. Thanks!

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @cmmdesai - I've left a few suggestions inline plus the following summary:
- Add the bulkUpload relationship to the synchronizationJob entity.
- Rename synchronization-synchronizationjobbulkupload.md to _synchronization-bulkupload.md_.
- GAP: Document the new permissions in the [permissions reference : Synchronization permissions](https://learn.microsoft.com/en-us/graph/permissions-reference#synchronization-permissions).
- Document your changelog without replacing old records. Please refer to the authoring guidance for the process of copying the changelog from the schema PR and adding it to your docs PR.

**Comment by cmmdesai** [comment]
@FaithOmbongi Thank you for the thorough review and feedback. I have incorporated the feedback and requested for another review.

File: changelog/Microsoft.IAM.json [hunk 1]
```diff
@@ -81,6 +81,32 @@
...
+          "Description": "Added the **bulkUpload** relationship to the [synchronizationJob](https://learn.microsoft.com/en-us/graph/api/resources/synchronization-synchronizationJob?view=graph-rest-beta) resource.",
+          "Target": "synchronizationJob"
+        }
+      ],
+      "Id": "a34189c5-af07-4671-9246-dde6c999e2dd",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2023-06-08T07:08:26.9619605Z",
+      "WorkloadArea": "Identity and access",
+      "SubArea": "Directory management"
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IAM.json:109]:
```suggestion
      "WorkloadArea": "Applications",
      "SubArea": ""
```

File: api-reference/beta/resources/synchronization-bulkupload.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+doc_type: resourcePageType 
+---
+
+# bulkUpload resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the capability of the Azure AD synchronization service to process bulk uploads.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/synchronization-bulkupload.md:16]:
```suggestion
Represents the capability of the Azure AD synchronization service to process bulk uploads. The bulk upload should be compliant with the [SCIM bulk request](https://www.rfc-editor.org/rfc/rfc7644.html#section-3.7) format.
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Minor changes

**Pull request merged**